build/build_scripts/expected_table_counts.txt

```diff
@@ -9,7 +9,7 @@
                             diagnosis        2
        diagnostic_selection_criterion        4
             expression_quantification    18200
-                feature_specification       30
+                feature_specification      730
```

Comment:
730 does not seem to be the right number of feature specifications. Maybe feature values ...

Reply:
30 are from before this PR, I think cell fractions or something for 30 phenotypes. And this PR adds 1 more feature, so I think it is 31.

Reply:
I added a commit that updated the other record counts as well (data analysis study, etc.).